pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
the issue is that ``klass`` is used in the shared docs for ``describe`` (and for ``pct_change``). This is ok if its actually defined in Series/DataFrame (it is defined in generic as well, but actually it is ``NDFrame``) which is kind of confusing.